_data/jobs.yml

```diff
@@ -1,29 +1,29 @@
-- title: Ruby Developer (m/f/d)
-  link: https://epages-gmbh.jobs.personio.de/job/202168?language=en
-  description: With your skills in Ruby, Rails, JavaScript, and PostgreSQL you are responsible for the development of our crucial in-house tools.
-  location: Bilbao
+- title: Frontend Developer (m/f/d)
```

Comment:
Just an idea: We could also add "HTML/CSS" here to use the full title (as we also do for the Tech Writer position) and give them a better idea of what the job is about :)

Reply:
Yeah I've also seen that line break when trying it out. It's not nice but I think the extra content might really help to attract people. So as long as we have another job in the overview that also needs two lines (which the Tech Writer job does), I'd add it. If this would be the only job with a two-line title, I'd remove it again :D